src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -1246,37 +1247,68 @@ class StartSessionHandler : public security_manager::SecurityManagerListener {
 #endif  // ENABLE_SECURITY
 
 RESULT_CODE ProtocolHandlerImpl::HandleControlMessageStartSession(
```

Comment:
Yes, done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/0a616eeda041c65ca9e35027f07898b763280ab1.